src/Saritasa.Tools.Common/Utils/EnumUtils.cs

```diff
@@ -18,14 +19,22 @@ public static class EnumUtils
     {
 #if NET40 || NET452 || NET461
         /// <summary>
-        /// Gets the value of Description attribute.
+        /// Gets a description of enum value. If <see cref="DescriptionAttribute"/> is specified for it, its value will be returned.
         /// </summary>
-        /// <param name="target">Enum.</param>
-        /// <returns>Description text.</returns>
+        /// <param name="target">Enum value.</param>
+        /// <returns>Description of the value.</returns>
         public static string GetDescription(Enum target)
         {
             var descAttribute = GetAttribute<DescriptionAttribute>(target);
-            return descAttribute?.Description;
+            if (descAttribute != null)
+            {
+                return descAttribute.Description;
+            }
+
+            var value = target.ToString();
+
+            // Split the value with spaces if it is intercapped.
+            return Regex.Replace(value, @"((?<=[a-z])([A-Z])|(?<=[A-Z])([A-Z][a-z]))", " $1", RegexOptions.Compiled);
```

Comment:
It does not make sense to use RegexOptions.Compiled, if you create a new object every time. Need to make the Regex static field.